code/__defines/materials.dm

```diff
@@ -14,7 +14,7 @@
 #define MAT_WOOD			"wood"
 #define MAT_LOG				"log"
 #define MAT_SIFWOOD			"alien wood"
```

Comment:
With changing alien log to sifwood log, wouldn't this want to be changed to sivian wood?

Reply:
this can be a followup tbh